### DISCUSSION TITLE
Use proper models with Pydantic instead of dicts

### DISCUSSION THREAD

**Discussion by rrooij** [terminusdb/terminusdb-client-python#348] [Ideas]
All Client functions return dicts now. It would be nice for the quality of the client if they would be proper classes that could be typechecked.

We could use Pydantic as well for extra validation options or schema creation.

**Comment by majidaldo** [comment]
related #362 , or just maintain a good bidirectional json schema (or LinkedML) conversion. from there you can do the programming language mapping.
